apps/api-documenter/src/yaml/YamlDocumenter.ts

```diff
@@ -397,6 +413,27 @@ export class YamlDocumenter {
     }
   }
 
+  // Prepends a string such as "/** @sealed @override */" to an item signature where appropriate.
+  private _formatCommentedAnnotations(signature: string, apiItem: ApiItem): string {
+    const apiItemTags: { isSealed?: boolean, isVirtual?: boolean, isOverride?: boolean }
+      = apiItem as any; // tslint:disable-line:no-any
```

Comment:
>any [](start = 19, length = 3)

You can use the `|` type concatenator here.

Reply:
Good point, fixed

---
In reply to: [188124681](https://github.com/Microsoft/web-build-tools/pull/655#discussion_r188124681) [](ancestors = 188124681)